webapps/frontend/ui/admin/client/scripts/pages/execution-metrics.js

```diff
@@ -17,190 +17,358 @@
 
 'use strict';
 
-var template = require('./execution-metrics.html?raw');
-var CamSDK = require('camunda-bpm-sdk-js/lib/angularjs/index');
+const template = require('./execution-metrics.html?raw');
```

Comment:
🙃  This file become quite bloated I would consider breaking it out into smaller components / helper methods but I also don't have a good idea how.

Reply:
I'll create a follow up ticket for that, it is somewhat big and I see some parts that could be moved easily.

Reply:
https://github.com/camunda/camunda-bpm-platform/issues/4153